x/wasm/keeper/contract_keeper.go

```diff
@@ -23,6 +23,18 @@ type decoratedKeeper interface {
 		authZ AuthorizationPolicy,
 	) (sdk.AccAddress, []byte, error)
 
+	createAndInstantiate(
```

Comment:
No need to optimize for this worflow. Let's stick with the existing create and instantiate methods when used as a library